src/Accelerators/NNPA/src/Conversion/ZHighToZLow/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(LLVM_TARGET_DEFINITIONS ZHighToZLow.td)
+mlir_tablegen(ZHighToZLow.inc -gen-rewriters "-I${DLC_ONNX_MLIR_SRC_ROOT}" "-I${DLC_SRC_ROOT}")
```

Comment:
Not sure what you are asking here?

Reply:
There is a function that invokes `mlir_tablegen` for you in MLIR.cmake. Can you use it here instead of directly invoking `mlir_tablegen` and having to do all the steps around it yourself?